EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,1118 @@
+---
+EIP: TBD
+
+Title: Royalty Bearing NFTs
+
+Description: This proposal outlines the structure of an extension of the ERC721 standard for NFTs to correctly define, process, and pay (hierarchical) onchain royalties from NFT sales, and goes beyond [EIP-2981](https://eips.ethereum.org/EIPS/eip-2981).
+
+Author: Andreas Freund (@Therecanbeonlyone1969) 
+
+discussions-to: https://ethereum-magicians.org/t/royalty-bearing-nfts/8453
+
+Status: Draft
+
+Type: Standards Track
+
+Category: ERC
+
+Created: 2022-03-14
+
+Requires: ERC721, ERC165, ERC2771
+---
+
+## Abstract
+The proposal directly connects NFTs and royalties in a smart contract architecture extending the ERC721 standard, with the aim of precluding central authorities from manipulating or circumventing payments to those who are legally entitled to them.
+
+The proposal builds upon the [OpenZeppelin Smart Contract Toolbox](https://github.com/OpenZeppelin/openzeppelin-contracts) architecture, and extends it to include royalty account management (CRUD), royalty balance and payments management, simple trading capabilities -- Listing/De-Listing/Buying -- and capabilities to trace trading on exchanges. The royalty management capabilities allow for hierarchical royalty structures, referred to herein as royalty trees, to be established by logically connecting a "parent" NFT to its "children", and recursively enabling NFT "children" to have more children. 
+
+## Motivation
+The management of royalties is an age-old problem characterized by complex contracts, opaque management, plenty of cheating and fraud. 
+
+The above is especially true for a hierarchy of royalties, where one or more assets is derived from an original asset such as a print from an original painting, or a song is used in the creation of another song, or distribution rights and compensation are managed through a series of affiliates. 
+
+In the example below, the artist who created the original is eligible to receive proceeds from every sale, and resale, of a print. 
+
+![Fig1](https://i.imgur.com/Py6bYQw.png)
```

Comment:
Figures should be added to `../assets/eip-4910/*` and be linked with a relative link.